django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -69,32 +74,57 @@ Requires core.js and SelectBox.js.
             filter_input.id = field_id + '_input';
 
             selector_available.appendChild(from_box);
-            const choose_all = quickElement('a', selector_available, gettext('Choose all'), 'title', interpolate(gettext('Click to choose all %s at once.'), [field_name]), 'href', '#', 'id', field_id + '_add_all_link');
-            choose_all.className = 'selector-chooseall';
+            const choose_all = quickElement(
+                'a',
+                selector_available,
+                interpolate(gettext('Choose all %s'), [field_name]),
+                'href', '#',
+                'role', 'button',
+                'id', field_id + '_add_all_link',
+                'class', 'selector-chooseall'
+            );
```

Comment:
There are a few "links" which are really buttons, if we can update the styling so that we can have these as proper buttons (so using the button tag and remove href # and role button) - that would be great
But that's more of a "bonus points" job and I imagine updating the css might be tricky